files/build_templates/buffers_config.j2

```diff
@@ -111,7 +111,7 @@ def
 {
     "CABLE_LENGTH": {
         "AZURE": {
-    {% for port in PORT %}
+    {% for port in PORT_ALL %}
```

Comment:
this is the correct change. in some cases, PORT is not defined. When we try to set the PORT cable length, we should use PORT_ALL instead of PORT.

Reply:
What cases do have the PORT is not defined? Should we fix the cases?

Reply:
when the platform does not have port_config.ini

according to code. get_port_config in minigraph.py.

Reply:
It's confusing to have both `PORT` and `PORT_ALL`. It shouldn't block this PR, but I feel like we should develop a cleaner solution. One for the backlog.

Reply:
PORT_ALL would always have a port list, even when PORT is not defined.